arclightning_backend/src/main.rs

```diff
@@ -115,14 +115,23 @@ mod test {
         let games: HashMap<String, Game> = toml::from_str(&games_toml).unwrap();
 
         // serialize as json
-        let json_object = serde_json::to_string(&games).unwrap();
-
-        // test json
-        // Sorry it's so gross lol
-        let mut test_json = r#"{"touhou_123":{"name":"Touhou","description":"bullet hell with waifus","genre":["bullet hell","anime"],"thumbnail_path":"path/to/touhou/thumbnail","exe_path":"C:\\Users\\THISUSER\\TOUHOU_PATH"},"melty_blood":{"name":"Melty Blood","description":"fighter with waifus","genre":["fighter","anime","2d"],"thumbnail_path":"path/to/melty_blood/thumbnail","exe_path":"C:\\Users\\THISUSER\\MELTY_BLOOD_PATH"}}"#;
-
-        // assert
-        assert_eq!(json_object, test_json);
+        let json_object_touhou = serde_json::to_string(&games.get("touhou_123")).unwrap();
+        let json_object_melty_blood = serde_json::to_string(&games.get("melty_blood")).unwrap();
+
+        // test cases separately to get around the nondeterministic order for hashmap 
+        let mut test_json_touhou = "{\"name\":\"Touhou\",\
```

Comment:
You should be able to use the ```rust r#"string"with"quotations"#" format so that you don't have to escape everything. My eyes are burning from all the backslashes

Reply:
Does this allow us to skip whitespace though?